docs/api/ShallowWrapper/props.md

```diff
@@ -16,6 +16,6 @@ expect(wrapper.props().foo).to.equal(10);
 
 #### Related Methods
 
-- [`.prop() => Object`](prop.md)
+- [`.prop(key) => Any`](prop.md)
```

Comment:
both are actually valid, and the argumentless form is more important anyways. Perhaps we should list both here?